rkt/image/dockerfetcher.go

```diff
@@ -51,6 +51,9 @@ type dockerFetcher struct {
 func (f *dockerFetcher) GetHash(u *url.URL) (string, error) {
 	ensureLogger(f.Debug)
 	dockerURL := d2acommon.ParseDockerURL(path.Join(u.Host, u.Path))
+	if dockerURL == nil {
+		return "", fmt.Errorf("Invalid docker URL")
```

Comment:
We [don't capitalize error strings](https://github.com/golang/go/wiki/CodeReviewComments#error-strings).

Also, maybe we can help the user a bit more by showing the valid syntax:

`fmt.Errorf("invalid docker URL %q. Syntax docker://[REGISTRYURL/]IMAGE_NAME[:TAG]", u)`